src/plugins/data/common/index_patterns/index_patterns/types.ts

```diff
@@ -33,3 +33,16 @@ export interface TypeMeta {
   aggs?: Record<string, AggregationRestrictions>;
   [key: string]: any;
 }
+
+export interface IndexPatternSpec {
+  _id: string;
```

Comment:
why the underscores ?

Reply:
this code is abstracted from existing code. You're right, it doesn't make sense, I'll remove them.